README.md

```diff
@@ -62,10 +62,20 @@ Stratisd requires Rust 1.17+ and Cargo to build. These may be available via
 your distribution's package manager. If not, [Rustup](https://www.rustup.rs/)
 is available to install and update the Rust toolchain.
 
+To submit pull requests, proper formatting is needed. To ensure this, 
```

Comment:
Alternatively:

""""
stratisd makes use of rustfmt to enforce consistent formatting in Rust files. PRs must pass the rustfmt task in the CI in order to be merged. The rustfmt task currently uses the specific rustfmt version 0.8.3. This can be installed by:

```
cargo install --vers 0.8.3 rustfmt
```
""""

Because you don't really need proper formatting to _submit_ a PR :)